minichain/contract.py

```
@@ -45,6 +45,17 @@ def _safe_exec_worker(code, globals_dict, context_dict, result_queue, gas_limit)
         except (OSError, ValueError) as e:
             logger.warning("Failed to set resource limits: %s", e)
 
+        transfers = []
+        
+        def transfer_out(address, amount):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `transfer_out`

Add return type annotation: `None`

(ANN202)